docs/postgres.md

```diff
@@ -29,3 +26,11 @@ LND is configured for Postgres through the following configuration options:
   database, user and password.
 * `db.postgres.timeout=...` to set the connection timeout. If not set, no
   timeout applies.
+
+Example as follows:
+```
+[db]
+db.backend=postgres
+db.postgres.dsn=postgresql://dbuser:dbpass@127.0.0.1:5432/dbname
+db.postgres.timeout=20s
```

Comment:
Maybe it is better to not set a timeout for postgres in the example. If the database is slow for whatever reason and a timeout is hit, there may be unexpected consequences in some cases? Thinking about a channel state update that cannot go through and causes a link to shut down.